crypto/objects/objects.txt

```diff
@@ -493,6 +493,90 @@ id-pkix-mod 13		: id-mod-timestamp-protocol
+id-pkix-mod 31		: id-mod-qualified-cert
+id-pkix-mod 32		: id-mod-crmf2003
+id-pkix-mod 33		: id-mod-pkix1-rsa-pkalgs
+id-pkix-mod 34		: id-mod-cert-bundle
+id-pkix-mod 35		: id-mod-qualified-cert-97
+id-pkix-mod 36		: id-mod-crmf2005
+id-pkix-mod 37		: id-mod-wlan-extns2005
+id-pkix-mod 38		: id-mod-sim2005
+id-pkix-mod 39		: id-mod-dns-srv-name-88
+id-pkix-mod 40		: id-mod-dns-srv-name-93
+id-pkix-mod 41		: id-mod-cmsContentConstr-88
+id-pkix-mod 42		: id-mod-cmsContentConstr-93
+id-pkix-mod 43		: id-mod-pkixCommon
+id-pkix-mod 44		: id-mod-pkixOtherCerts
+id-pkix-mod 45		: id-mod-pkix1-algorithms2008
+id-pkix-mod 46		: id-mod-clearanceConstraints
+id-pkix-mod 47		: id-mod-attribute-cert-02
+id-pkix-mod 48		: id-mod-ocsp-02
+id-pkix-mod 49		: id-mod-v1AttrCert-02
+id-pkix-mod 50		: id-mod-cmp2000-02
+id-pkix-mod 51		: id-mod-pkix1-explicit-02
+id-pkix-mod 52		: id-mod-scvp-02
+id-pkix-mod 53		: id-mod-cmc2002-02
+id-pkix-mod 54		: id-mod-pkix1-rsa-pkalgs-02
+id-pkix-mod 55		: id-mod-crmf2005-02
+id-pkix-mod 56		: id-mod-pkix1-algorithms2008-02
+id-pkix-mod 57		: id-mod-pkixCommon-02
+id-pkix-mod 58		: id-mod-algorithmInformation-02
+id-pkix-mod 59		: id-mod-pkix1-implicit-02
+id-pkix-mod 60		: id-mod-pkix1-x400address-02
+id-pkix-mod 61		: id-mod-attribute-cert-v2
+id-pkix-mod 62		: id-mod-sip-domain-extns2007
+id-pkix-mod 63		: id-mod-cms-otherRIs-2009-88
+id-pkix-mod 64		: id-mod-cms-otherRIs-2009-93
+id-pkix-mod 65		: id-mod-ecprivatekey
+id-pkix-mod 66		: id-mod-ocsp-agility-2009-93
+id-pkix-mod 67		: id-mod-ocsp-agility-2009-88
+id-pkix-mod 68		: id-mod-logotype-certimage
+id-pkix-mod 69		: id-mod-pkcs10-2009
+id-pkix-mod 70		: id-mod-dns-resource-record
+id-pkix-mod 71		: id-mod-send-cert-extns
+id-pkix-mod 72		: id-mod-ip-addr-and-as-ident-2
+id-pkix-mod 73		: id-mod-wlan-extns-2
+id-pkix-mod 74		: id-mod-hmac
+id-pkix-mod 75		: id-mod-enrollMsgSyntax-2011-88
+id-pkix-mod 76		: id-mod-enrollMsgSyntax-2011-08
+id-pkix-mod 77		: id-mod-pubKeySMIMECaps-88
+id-pkix-mod 78		: id-mod-pubKeySMIMECaps-08
+id-pkix-mod 79		: id-mod-dhSign-2012-88
+id-pkix-mod 80		: id-mod-dhSign-2012-08
+id-pkix-mod 81		: id-mod-ocsp-2013-88
+id-pkix-mod 82		: id-mod-ocsp-2013-08
+id-pkix-mod 83		: id-mod-TEST-certPolicies
+id-pkix-mod 84		: id-mod-bgpsec-eku
+id-pkix-mod 85		: id-mod-nai-realm-08
+id-pkix-mod 86		: id-mod-tls-feature-2015
+id-pkix-mod 87		: id-mod-EST-Alt-Challenge
+id-pkix-mod 88		: id-mod-mudURLExtn2016
+id-pkix-mod 89		: id-mod-tn-modul
+id-pkix-mod 90		: id-mod-ip-addr-and-as-ident-v2
+id-pkix-mod 91		: id-mod-ip-addr-and-as-ident-2v2
+id-pkix-mod 92		: id-mod-lamps-eai-addresses-2016
+id-pkix-mod 93		: id-mod-safecurves-pkix
+id-pkix-mod 94		: id-mod-pkix1-shakes-2019
+id-pkix-mod 95		: id-mod-delegated-credential-extn
+id-pkix-mod 96		: id-mod-MASAURLExtn2016
+id-pkix-mod 97		: id-mod-anima-acpnodename-2020
+id-pkix-mod 98		: id-mod-est-2019
+id-pkix-mod 99		: id-mod-cmp2021-88
+id-pkix-mod 100		: id-mod-cmp2021-02
```

Comment:
Indentation??

Reply:
This apparent extra indentation is an artifact of the `git diff` output due to the added `+`, which caused the tab following the `100` to jump in the diff output. In the editor it is fine.
BTW, in the automatic insertion I used the same two tabs per line that were present for the existing lines before.